api/v1alpha1/schemas.cue

```diff
@@ -60,7 +60,7 @@ package v1alpha1
 	metadata!:  #metadata
 	spec: {
 		// Owners is the list of identifiers of owners of the project.
-		// It can be any string that uniquely identifies the owners, such as email addresses or Jac group identifiers.
+		// It can be any string that uniquely identifies the owners, such as email addresses or backstage group names
```

Comment:
Is the plan to wait for backstage to be available before merging this?

Reply:
Nope, the field is now an array of arbitrary strings, there is no actual connection to backstage, its really just a "suggested use".